src/index.html

```diff
@@ -60,31 +64,307 @@
 
     <main class="catalog">
       <div class="catalog__card">
-        Product card
+        <img
+            src="/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="catalog__img"
+          >
+        <h2 class="catalog__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="catalog__code">
+          Product code: 195434
+        </h3>
+        <div class="catalog__rate-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="catalog__rate-count">Reviews: 5</div>
+        </div>
+        <div class="catalog__price-container">
+          <div class="catalog__price-label">
+            Price:
+          </div>
+          <div class="catalog__price">
+            $2,199
+          </div>
+        </div>
+        <a
+          href="/"
+          class="catalog__buy"
+        >
+          buy
+        </a>
       </div>
 
       <div class="catalog__card">
-        Product card
+        <img
+            src="/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="catalog__img"
+          >
```

Comment:
Pay attention on indents. Fix other place too
```suggestion
        <img
          src="/images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
          class="catalog__img"
        >
```